monitor/app/metrics.go

```diff
@@ -51,10 +51,31 @@ var (
 		Help:      "Suggested OmniEVM gas tip cap in gwei",
 	})
 
-	inflationAvg = promauto.NewGauge(prometheus.GaugeOpts{
+	rewardsAvg = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
lets move this to `staking` package.

Then metrics can be prefixed with `monitor_staging_rewards_*`